### ISSUE TITLE
Using https:// failed on Ubuntu 17.04, might want to update README to git://

### ISSUE THREAD

**Issue by michaelpb** [pydanny/cookiecutter-djangopackage#253]
Minor thing, fix + suggestion covered in subject line, almost certainly unrelated to `cookiecutter-djangopackage`, but the fix is like 4 characters in documentation so it might be worth it if others encounter this issue.

Full bug report as follows:

---

* Date you used Cookiecutter DjangoPackage: Oct 30 2017
* Cookiecutter version used, if known: 1.5.1
* Python version: 3.5
* Operating System: Ubuntu 17.04

### Description

Very simple fix to this. Change the cmd in the readme to this:

```
cookiecutter git://github.com/pydanny/cookiecutter-djangopackage.git
```

This is because the command in the README failed for me, while the above worked:

```
michaelb@icheb:~/projects$ cookiecutter https://github.com/pydanny/cookiecutter-djangopackage.git
Traceback (most recent call last):
  File "/home/michaelb/.local/lib/python3.5/site-packages/cookiecutter/vcs.py", line 123, in clone
    stderr=subprocess.STDOUT,
  File "/usr/lib/python3.5/subprocess.py", line 316, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 398, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'clone', 'https://github.com/pydanny/cookiecutter-djangopackage.git']' returned non-zero exit status 128

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/michaelb/.local/bin/cookiecutter", line 11, in <module>
    sys.exit(main())
  File "/home/michaelb/.local/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/michaelb/.local/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/michaelb/.local/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/michaelb/.local/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/michaelb/.local/lib/python3.5/site-packages/cookiecutter/cli.py", line 123, in main
    default_config=default_config,
  File "/home/michaelb/.local/lib/python3.5/site-packages/cookiecutter/main.py", line 63, in cookiecutter
    no_input=no_input,
  File "/home/michaelb/.local/lib/python3.5/site-packages/cookiecutter/repository.py", line 84, in determine_repo_dir
    no_input=no_input,
  File "/home/michaelb/.local/lib/python3.5/site-packages/cookiecutter/vcs.py", line 132, in clone
    if 'not found' in clone_error.output.lower():
TypeError: a bytes-like object is required, not 'str'
```

**Comment by ssteinerx** [comment]
Will do, but looks like an upstream issue in [Cookiecutter](https://github.com/audreyr/cookiecutter) itself where https:// repo usage is documented.  I'll file one there with a reference to this one.

**Comment by pydanny** [comment]
Resolved in 4ec87d49c73b8e8537e42b671b5d3300e3f4f385
